h/templates/activity/search.html.jinja2

```diff
@@ -222,7 +221,7 @@
       <div class="search-result-sidebar__organization">
         <span class="search-result-sidebar__org-logo">
           {% if organization.logo %}
-            <img src="{{ organization.logo }}" alt="Organization logo" class='search-result-sidebar__logo'></img>
+            <img src="{{ organization.logo }}" alt="Organization logo" class='search-result-sidebar__logo'>
```

Comment:
Huh, I realize I'm still a robot when it comes to `img` tags and still close them (`/>`). But hey, you know, HTML5 has been around a while now and I should really get with the proverbial program :).